src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -633,8 +633,8 @@ export default function SharedView(): JSX.Element {
     try {
       if (shareItem.isRootLink) {
         const { credentials } = await shareService.getSharedFolderContent(
-          // folderUUID
-          shareItem.uuid,
+          // folderUUID = uuid. If it's a file, it comes as folderUuid.
+          shareItem.isFolder ? shareItem.uuid : shareItem.folderUuid,
```

Comment:
We should refactor this rather than patching it with this misleading naming @Ap1402